doc/start/documenting-ceph.rst

```diff
@@ -973,6 +973,16 @@ convention was preferred because it made the documents more readable in a
 command line interface. As of 2023, though, we have no preference for one over
 the other. Use whichever convention makes the text easier to read.
 
+Using a part of a sentence as a hyperlink, `like this <docs.ceph.com>`_, is
+discouraged. The convention of writing "See X" is preferred. Here are some
+preferred formulations:
+
+#. For more information, see `docs.ceph.com <docs.ceph.com>`_.
+
+#. See `docs.ceph.com <docs.ceph.com>`_.
+
```

Comment:
Is it 3 empty lines here on purpose and not 2 like usually?

Reply:
It's not on purpose. Blank lines, being unrendered, are not of great concern.

Reply:
My employer has GitLab commit hooks that beg to differ ;)

Reply:
Alright. I've removed the third blank line.